bigbluebutton-html5/imports/ui/components/actions-bar/create-breakout-room/component.jsx

```diff
@@ -0,0 +1,159 @@
+        1: value,
+      }),
+      freeJoin: this.state.freeJoin,
+      sequence: value,
+    }));
+
+    createBreakoutRoom(rooms, durationTime, true);
+  }
+
+  changeNumberOfRooms(event) {
+    this.setState({ numberOfRooms: Number.parseInt(event.target.value, 10) });
+  }
+
+  changeDurationTime(event) {
+    this.setState({ durationTime: Number.parseInt(event.target.value, 10) || '' });
+  }
+
+  increaseDurationTime() {
+    this.setState({ durationTime: (1 * this.state.durationTime) + 1 });
+  }
+
+  decreaseDurationTime() {
+    const number = ((1 * this.state.durationTime) - 1);
+    this.setState({ durationTime: number < 1 ? 1 : number });
+  }
+
+  render() {
+    const { intl } = this.props;
+    return (
+      <Modal
+        title={intl.formatMessage(intlMessages.breakoutRoomTitle)}
+        confirm={
+          {
+            label: intl.formatMessage(intlMessages.confirmButton),
+            callback: this.onCreateBreakouts,
+          }
+        }
+      >
+        <div className={styles.content}>
+          <p className={styles.subTitle}>
+            {intl.formatMessage(intlMessages.breakoutRoomDesc)}
+          </p>
+          <div className={styles.breakoutSettings}>
+            <label>
+              <p className={styles.labelText}>{intl.formatMessage(intlMessages.numberOfRooms)}</p>
+              <select name="numberOfRooms" className={styles.inputRooms} value={this.state.numberOfRooms} onChange={this.changeNumberOfRooms}>
+                {
+                  _.range(MIN_BREAKOUT_ROOMS, MAX_BREAKOUT_ROOMS + 1).map(item => (<option key={_.uniqueId('value-')}>{item}</option>))
+                }
+              </select>
+            </label>
+            <label >
+              <p className={styles.labelText}>{intl.formatMessage(intlMessages.duration)}</p>
+              <div className={styles.durationArea}>
+                <input type="number" className={styles.duration} min={MIN_BREAKOUT_ROOMS} value={this.state.durationTime} onChange={this.changeDurationTime} />
+                <span>
+                  <HoldButton
+                    key="decrease-breakout-time"
+                    exec={this.decreaseDurationTime}
+                    minBound={MIN_BREAKOUT_ROOMS}
+                    value={this.state.durationTime}
+                  >
+                    <Icon
+                      className={styles.iconsColor}
+                      iconName="substract"
+                    />
+                  </HoldButton>
+                  <HoldButton
+                    key="increase-breakout-time"
+                    exec={this.increaseDurationTime}
+                  >
+                    <Icon
+                      className={styles.iconsColor}
+                      iconName="add"
+                    />
+                  </HoldButton>
+
+                </span>
+              </div>
+            </label>
+            <p className={styles.randomText}>{intl.formatMessage(intlMessages.randomlyAssign)}</p>
```

Comment:
This will turn in a button in a later PR? Or which is the purpose of this label?

Reply:
Yes, this will be a button on next PR.